cts/update/field/max.json

```diff
@@ -0,0 +1,61 @@
+{
+  "1-number": {
+    "request": {
+      "update": "values",
+      "updates": [
+        {
+          "q": {
+            "_id": "int32"
+          },
+          "u": {
+            "$max": {
+              "v": {
+                "$numberInt": "50"
```

Comment:
In CTS, how do we plan to test different test cases such as `{$max: {v: 20}}` and v is left as 42? Possibly we are currently testing each simple case?

Reply:
Yes, currently we test for the simplest behavior, just to cover all operators. Later, we're going to test each field, and corner cases.